.github/settings.yml

```diff
@@ -0,0 +1,35 @@
+repository:
+  # See https://developer.github.com/v3/repos/#edit for all available settings.
+
+  # The name of the repository. Changing this will rename the repository
+  name: outbound-oss
+
+  # A short description of the repository that will show up on GitHub
+  description: Documentation and guidance for handling outbound open source for organizations
+
+  # A URL with more information about the repository
+  homepage: https://todogroup.org
+
+# See https://docs.github.com/en/rest/reference/teams#add-or-update-team-repository-permissions for available options
+teams:
+  - name: steering-committee
+    # The permission to grant the team. Can be one of:
+    # * `pull` - can pull, but not push to or administer this repository.
+    # * `push` - can pull and push, but not administer this repository.
+    # * `admin` - can pull, push and administer this repository.
+    # * `maintain` - Recommended for project managers who need to manage the repository without access to sensitive or destructive actions.
+    permission: admin
+
+# Collaborators: give specific users access to this repository.
+# See https://docs.github.com/en/rest/reference/collaborators for available options
+collaborators:
+  # maintain
+  - username: misappi
+    permission: maintain
+
+  # push
+  - username: cornelius
+    permission: push
+
+  - username: mkurzman
+    permission: push
```

Comment:
@justaugustus Could you please add @OliverFendt as a collaborator? 🙂 
```
- username: oliverfendt
  permission: push
```

Reply:
Also, all these contributors (@misappi @cornelius @mkurzman and @OliverFendt ) should have merge permissions. Not sure if `push` permission also allows merging PRs in this repo, or should be upgraded to `maintain`

Reply:
> Also, all these contributors (@misappi @cornelius @mkurzman and @OliverFendt ) should have merge permissions. Not sure if `push` permission also allows merging PRs in this repo, or should be upgraded to `maintain`

`push` is sufficient for write access, but I've upgraded everyone to `maintain` to allow maintenance of the repo w/o too much SC oversight.

> @justaugustus Could you please add @OliverFendt as a collaborator? 🙂

He wasn't listed before (see below), but I've added him now!

<img width="803" alt="Screen Shot 2022-01-18 at 07 36 27" src="https://user-images.githubusercontent.com/567897/149938862-0ccc99a2-4b1c-469b-a10b-336f468064a9.png">